backend/tournesol/admin.py

```diff
@@ -197,17 +217,27 @@ class PollAdmin(admin.ModelAdmin):
         'algorithm',
         'entity_type',
         'get_n_criterias',
+        'get_n_comparisons',
+        'get_n_comparisons_per_criteria',
     )
     list_filter = (
         'algorithm',
         'entity_type'
     )
     inlines = (CriteriasInline,)
 
-    @admin.display(description="Nb of criterias")
+    @admin.display(description="# criterias")
     def get_n_criterias(self, obj):
         return obj.criterias.count()
 
+    @admin.display(description="# comparisons")
+    def get_n_comparisons(self, obj):
+        return obj.comparisons.count()
```

Comment:
Not sure, because it has been a long time since I've worked with the administration interface, but this line may generate an SQL query for each object displayed in the list.

I'm going to check.